Jenkinsfile

```diff
@@ -81,7 +81,7 @@ pipeline {
 									# rm -rf ~/.lisk/
 									# install -D tests/dev_config_and_db/genesis_block.json ~/.lisk/lisk-core/config/devnet/genesis_block.json
 									# ./lisk-core/bin/lisk-core blockchain:import --force tests/dev_config_and_db/tokens_unlocked_dev_blockchain.db.tar.gz
-									# ./lisk-core/bin/lisk-core forger-info:import --force tests/dev_config_and_db/forger.db.tar.gz
+									# ./lisk-core/bin/lisk-core generator-info:import --force tests/dev_config_and_db/generator.db.tar.gz
```

Comment:
@oskarleonard While this was renamed, the generator file was not renamed accordingly

Reply:
Fixed